BUILD.md

```diff
@@ -42,3 +42,17 @@ The progress of the build can be monitored with:
 Once built, the image will be visible in the Image Stream created by the same command:
 
         oc describe imagestream openshift-ansible
+
+## Build the System Container
```

Comment:
a system container doesn't run only on Atomic Host.  Is it fine to specify what the differences are and add a link to http://www.projectatomic.io/blog/2016/09/intro-to-system-containers/ ?